pkg/volume/local/local_test.go

```diff
@@ -57,6 +60,44 @@ func getPlugin(t *testing.T) (string, volume.VolumePlugin) {
 	return tmpDir, plug
 }
 
+func getBlockPlugin(t *testing.T) (string, volume.BlockVolumePlugin) {
+	tmpDir, err := utiltesting.MkTmpdir("localVolumeTest")
+	if err != nil {
+		t.Fatalf("can't make a temp dir: %v", err)
+	}
+	fakeDev := path.Join(tmpDir, testDev)
+	file, err := os.OpenFile(fakeDev, os.O_RDONLY|os.O_CREATE, 0666)
+	if err != nil {
+		t.Fatalf("Cannot create fakedev: %v", err)
+	}
+	err = file.Close()
+	if err != nil {
+		t.Fatalf("Cannot close fakedev: %v", err)
+	}
+	podPath := path.Join(tmpDir, testPodPath)
+	err = os.MkdirAll(path.Dir(podPath), 0766)
+	if err != nil {
+		t.Fatalf("Cannot create directories for %q: %v", path.Dir(podPath), err)
+	}
+
+	err = os.Symlink(fakeDev, podPath)
+	if err != nil {
+		t.Fatalf("Failed to create symlink for %q: %v", podPath, err)
+	}
+
+	plugMgr := volume.VolumePluginMgr{}
+	plugMgr.InitPlugins(ProbeVolumePlugins(), nil /* prober */, volumetest.NewFakeVolumeHost(tmpDir, nil, nil))
+	plug, err := plugMgr.FindMapperPluginByName(localVolumePluginName)
+	if err != nil {
+		os.RemoveAll(tmpDir)
+		t.Fatalf("Can't find the plugin by name")
```

Comment:
nit: Can you please print the name of the plugin, that failed to be found?

Reply:
Fixed.